application/ui/src/features/inference/sinks/output-formats.component.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Checkbox, CheckboxGroup } from '@geti/ui';
+
+import { OutputFormat } from './utils';
+
+import classes from './output-formats.module.scss';
+
+export const OutputFormats = () => {
+    return (
+        <CheckboxGroup label='Output Formats' UNSAFE_className={classes.checkboxGroup}>
+            <Checkbox name='output_formats' value={OutputFormat.PREDICTIONS}>
```

Comment:
all 3 checkboxes have the same `name`

Reply:
Yes, it will return an array with all the selected options